packages/replay/src/constants.ts

```diff
@@ -14,9 +14,6 @@ export const UNABLE_TO_SEND_REPLAY = 'Unable to send Replay';
 // The idle limit for a session
 export const SESSION_IDLE_DURATION = 300_000; // 5 minutes in ms
 
-// Grace period to keep a session when a user changes tabs or hides window
-export const VISIBILITY_CHANGE_TIMEOUT = SESSION_IDLE_DURATION;
-
```

Comment:
Can we keep this in case we want to change this value in the future, or at least split it out into a different PR.

Reply:
I can keep both for now, just figured it's a bit of byte overhead that we aren't actually using right now, and not sure how probable it is that we're going to start using a different value there? 🤔 I can def. do this in a separate PR though to keep this clearer.

Reply:
Yeah I'm OK to remove it, but let's do it in a separate PR.